app/utils/wikiFetch.tsx

```diff
@@ -52,5 +52,14 @@ export async function getWikipediaContestantData(): Promise<IWikipediaContestant
         return aContestant
     }).toArray()
 
-    return contestantData
+    const result = filterEmptyContestants(contestantData)
+
+    return result
+}
+
+export function filterEmptyContestants(contestantList: IWikipediaContestantData[]): IWikipediaContestantData[] {
+    // I believe that the behavior that led to this addition is missing all
+    // properties but the name seemed like the most important one for now
+    return contestantList.filter(x =>  x.name != null && x.name !== "")
```

Comment:
I can prove this checks works via the tests, but I have no idea if this is the canonical way to do this kind of thing 😅

Reply:
I believe  `contestantList.filter(x => x.name);`  would also filter out names that are `null` or empty strings.